.circleci/config.yml

```diff
@@ -47,16 +47,24 @@ jobs:
           root: .
           paths:
             - .
-  lint_backend_and_frontend:
```

Comment:
I think you'll need to remove this job on line 101 in this file and add `lint_backend` and `lint_backend` jobs